io_scene_godot/converters/utils.py

```diff
@@ -152,13 +153,19 @@ def to_mesh(self, triangulate=True, preserve_vertex_groups=True,
         # mesh result can be none if the source geometry has no faces, so we
         # need to consider this if we want a robust exporter.
         if mesh is not None:
-            if triangulate:
-                triangulate_mesh(mesh)
-
             self.has_tangents = bool(mesh.uv_layers) and bool(mesh.polygons)
             if calculate_tangents:
                 if self.has_tangents:
-                    mesh.calc_tangents()
+                    try:
+                        mesh.calc_tangents()
+                    except RuntimeError:
+                        # Mesh must have n-gons
+                        logging.info(
```

Comment:
I wasn't sure if I should use a warning since it's much more visible. I updated it and also tweaked the message slightly for more friendliness, see if it's okay.